### PR TITLE
Feature/detail UI

### PR DESCRIPTION
**Pull request by at-hanhpham**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Meete/View/NewItemCell/NewItemCell.swift [hunk 1]
```diff
@@ -50,11 +73,16 @@ extension NewItemCell: UICollectionViewDelegateFlowLayout {
 // MARK: - UICollectionViewDataSource
 extension NewItemCell: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return 10
+        return listItem.count
```
Comment (at-lamle) [inline Meete/View/NewItemCell/NewItemCell.swift]: Please use another variable

File: Meete/View/NewItemCell/NewItemCell.swift [hunk 2]
```diff
@@ -11,10 +11,33 @@ import UIKit
...
 
+    weak var delegate: NewItemCellDelegate?
+    @IBOutlet private weak var titleLabel: UILabel!
     @IBOutlet private weak var collectionView: UICollectionView!
+    @IBOutlet private weak var styleCellImage: UIImageView!
+    fileprivate var listItem: [AdvertisementItem] = []
     static let height: CGFloat = 270 * Ratio.horizontal
+    
+    struct CellResult {
+        var listItem: [AdvertisementItem]?
```
Comment (at-lamle) [inline Meete/View/NewItemCell/NewItemCell.swift]: Please edit name function

File: Meete/View/SlideCell/SlideCell.swift [hunk 3]
```diff
@@ -11,6 +11,8 @@ import UIKit
 class SlideCell: UITableViewCell {
 
     @IBOutlet private weak var collectionView: UICollectionView!
+    var galleryPaths: [String]?
```
Comment (at-lamle) [inline Meete/View/SlideCell/SlideCell.swift]: Please edit galleryPaths name

File: Meete/Controllers/Detail/DetailViewController.swift [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    @objc private func pressFavoriteButton(_ button: UIButton) {
+        button.isSelected = !button.isSelected
+        item.isFavorite = !item.isFavorite
+    }
+}
+
+// MARK: - AddressCellDelegate
+extension DetailViewController: AddressCellDelegate {
+    func addressCell(_ addressCell: AddressCell, selectedButton: UIButton) {
+        let directionVC = DirectionViewController(destination: item.location)
```
Comment (at-hanhpham) [inline Meete/Controllers/Detail/DetailViewController.swift]: item is a variable of DetailViewController

**Pull request closed without merging**